docker-compose.yml

```diff
@@ -20,7 +20,7 @@ services:
       - "memcached"
 
   db:
-    image: postgres:14.2-alpine3.15
+    image: postgres:14.4-alpine3.16
```

Comment:
Is this completely stable?
In my experience,  `[3.16.0, 3.16, 3, latest](https://github.com/alpinelinux/docker-alpine/blob/c9515859c035acbb899c6738506954e36f57e7dc/x86_64/Dockerfile)[3.16.0, 3.16, 3, latest](https://github.com/alpinelinux/docker-alpine/blob/c9515859c035acbb899c6738506954e36f57e7dc/x86_64/Dockerfile)` (at this date, 3.16 seems to be latest) gives problems.

Reply:
It was released two months ago, and from what I've seen on DockerHub, all the popular images are starting to use this version for their alpine image version. But if we find any issues, we can downgrade later.